internal/pyxis/pyxis.go

```diff
@@ -430,16 +434,25 @@ func (p *pyxisClient) createTestResults(ctx context.Context, testResults *TestRe
 		return nil, fmt.Errorf("could not read body: %w", err)
 	}
 
+	bodyAsString := string(body)
```

Comment:
Nit: Not a big fan of including the type in the var name. Just call it `body`, rename the above to just `b`, and be done with it.

Reply:
I updated this to `respBody`, since we already have `b` and `resp` and `body` elsewhere in this method. LMK what you think.